packages/expo-cli/src/commands/eas-build/status/action.ts

```diff
@@ -104,11 +104,7 @@ function printBuildTable(builds: Build[]) {
       }).format(new Date(build.createdAt)),
       platform: build.platform,
       status: build.status.replace(/-/g, ' '),
-      artifact: buildUrl
```

Comment:
Will it show full URL without a column width?

Though I was thinking about an alternative format for status. The table format isn't great there's not a lot of horizontal space. We also might want to show the build ID since it's useful for reporting issues.

Reply:
yes, currently last column is the same size as the widest content

I tried to check with isSupprted if the URLs are supported in a terminal emulator, but it always returned true.

> The table format isn't great there's not a lot of horizontal space

We will in the future replace it with shortened url so it might improve the situation , build id is part of the url so I think it's good enough at least for now